### ISSUE TITLE
Admin UI linking a tag to a post in default example throws an error

### ISSUE THREAD

**Issue by jrmyio** [keystonejs/keystone#6465]
## Describe the bug

Linking an existing tag to a post in the default keystone-next example throws a graphql error when selecting it in the tag from the select field:

**Request payload:**
{"variables":{"ids":["ckt7762ds10138bcjx4fibihyq"]},"query":"query ($ids: [ID!]!) {\n  items: tags(where: {id_in: $ids}) {\n    name\n    id\n    __typename\n  }\n}\n"}

**Error Response**
{"errors":[{"message":"Field \"id_in\" is not defined by type \"TagWhereInput\". Did you mean \"id\"?","locations":[{"line":2,"column":23}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED","exception":{"stacktrace":["GraphQLError: Field \"id_in\" is not defined by type \"TagWhereInput\". Did you mean \"id\"?","    at ObjectField (/home/test/keystone/node_modules/graphql/validation/rules/ValuesOfCorrectTypeRule.js:73:29)","    at Object.enter (/home/test/keystone/node_modules/graphql/language/visitor.js:323:29)","    at Object.enter (/home/test/keystone/node_modules/graphql/utilities/TypeInfo.js:370:25)","    at visit (/home/test/keystone/node_modules/graphql/language/visitor.js:243:26)","    at Object.validate (/home/test/keystone/node_modules/graphql/validation/validate.js:69:24)","    at validate (/home/test/keystone/node_modules/apollo-server-core/src/requestPipeline.ts:536:14)","    at Object.<anonymous> (/home/test/keystone/node_modules/apollo-server-core/src/requestPipeline.ts:302:32)","    at Generator.next (<anonymous>)","    at fulfilled (/home/test/keystone/node_modules/apollo-server-core/dist/requestPipeline.js:5:58)","    at runMicrotasks (<anonymous>)","    at processTicksAndRejections (node:internal/process/task_queues:96:5)"]}}}]}

## To Reproduce

- Install keystone and go to the admin UI. 
- Create a Post and Create a Tag, then save the post.
- Then remove the tag from the post and Save Changes.
- Now try to select the tag in the input field again. This will throw a graphql error as shown above.

## Expected behaviour
It should not throw an error and add the tag to the post.

## Package.json
```json
{
  "name": "keystone-app",
  "version": "2.0.12",
  "private": true,
  "scripts": {
    "dev": "keystone-next dev",
    "start": "keystone-next start",
    "build": "keystone-next build",
    "postinstall": "keystone-next postinstall"
  },
  "dependencies": {
    "@keystone-next/auth": "^31.0.0",
    "@keystone-next/fields": "^14.0.0",
    "@keystone-next/fields-document": "^8.0.0",
    "@keystone-next/keystone": "^24.0.1",
    "typescript": "^4.3.5"
  }
}
```

Additionally, there seems to be no way to select an author inside the editing a post, it only shows *Create User*, but no `Link existing user`

**Comment by bladey** [comment]
Thanks @jrmyio, I can reproduce the first error.

In regards to the second, this is because inline connect isn't enabled for Users, as shown here:

<img width="481" alt="Screen Shot 2021-09-09 at 10 04 39 am" src="https://user-images.githubusercontent.com/737821/132601388-aa926346-ec30-4fbf-a6f6-98e8db9a9895.png">

**Comment by bladey** [comment]
**Further debugging:**

1) Selecting item to link in a relationship:

<img width="763" alt="Screen Shot 2021-09-10 at 10 05 57 am" src="https://user-images.githubusercontent.com/737821/132778137-254aaedc-fd52-4df8-af0a-3b72417df6bd.png">

2) This error gets thrown:

<img width="1913" alt="Screen Shot 2021-09-10 at 10 04 28 am" src="https://user-images.githubusercontent.com/737821/132778163-3440432c-d1c3-4d4b-8aba-29ffa9ac9eb1.png">

3) These are the available types for **TagWhereInput**:

<img width="426" alt="Screen Shot 2021-09-10 at 10 05 24 am" src="https://user-images.githubusercontent.com/737821/132778177-8b3f67ec-a5d8-4f17-84c6-a2f5cba0232d.png">

4) This is the code for the Admin UI where `id_in` is defined:

`/packages/keystone/src/fields/types/relationship/views/cards/index.tsx`

<img width="889" alt="image" src="https://user-images.githubusercontent.com/737821/132778357-d74afec3-c3cd-4d26-afa7-1e523151651d.png">

`/packages/keystone/src/fields/types/relationship/views/index.tsx`

<img width="724" alt="image" src="https://user-images.githubusercontent.com/737821/132778424-716a710c-6573-4a6c-8902-7c02b88ba535.png">

**Comment by bladey** [comment]
Hi @jrmyio, this is now resolved in `"@keystone-next/keystone": "25.0.3"` https://github.com/keystonejs/keystone/pull/6526.

**Comment by ngodacthien** [comment]
When i create the tag, the browser will refresh. But when i change ui : ishidden : false and add in TAG it will be normal and i can create tag in Post .
